hack/generate-kustomize-patches.sh

```diff
@@ -21,7 +21,7 @@ KUSTOMIZE_INPUT_DIR="$ROOT_DIR/config/helm/input"
 # Download upstream manifests
 helm_values="$HELM_DIR/values.yaml"
 # Giant Swarm specific, since we don't use GitHub releases in https://github.com/giantswarm/cluster-api-provider-aws
-version="$(yq e '.tag' "$helm_values")"
+version="$(yq e -e '.tag' "$helm_values")"
```

Comment:
These changes are unrelated and ensure that reading a non-existent value leads to an error.